llvm/lib/IR/Verifier.cpp

```diff
@@ -431,7 +435,10 @@ class Verifier : public InstVisitor<Verifier>, VerifierSupport {
 
     Broken = false;
     // FIXME: We strip const here because the inst visitor strips const.
-    visit(const_cast<Function &>(F));
+    {
+      llvm::TimeTraceScope domScope("Verifier visit");
```

Comment:
scope name copy-past-o?

Reply:
Changed, thanks for noting. There was another one below.